dashboard/views_test.py

```diff
@@ -93,13 +67,10 @@ def test_anonym_access(self):
 
     def test_get_dashboard(self):
         """Test for GET"""
-        res = self.get_with_mocked_refresh_backends()
-
-        assert res.status_code == status.HTTP_200_OK
-        program_ids = [program['id'] for program in res.data]
-        # not live and not enrolled programs are missing
-        assert program_ids == [self.program_1.pk, self.program_2.pk]
-        assert Program.objects.count() == 4
+        with patch('backends.utils.refresh_user_token', autospec=True), \
+                patch('dashboard.api.get_user_program_info', autospec=True):
```

Comment:
Please:
 - assert the arguments passed into each of these mocked functions
 - provide appropriate return values
 - assert the value of res.data

It is really important that we test the dashboard API well to keep regressions like this one from happening again.